src/run.ts

```diff
@@ -130,7 +133,9 @@ export async function runPublish({
     { cwd }
   );
 
-  await gitUtils.pushTags();
+  if (!commitUsingApi) {
```

Comment:
It would be great if we could leverage the strategy pattern or something so the code here wouldn't have to branch on `commitUsingApi` so much